content/js1/sprints/1/prep/index.md

```diff
@@ -8,4 +8,443 @@ backlog= 'Module-JS1'
 backlog_filter= 'Week 1'
 +++
 
+## Prerequisites
 
+- Basic arithmetic
+- Access to a Linux/MacOS terminal
+
+## Learning objectives
+
+- Define a terminal
+- Describe the purpose of VSCode
+- Define the Node REPL
+- Define an expression
+- Identify the syntactic features of a variable declaration
+- Explain why we use variable declarations
+- Describe what template literals do
+- Create strings using variables and template literals
+- Explain how we use basic arithmetical operators
+- Evaluate basic JavaScript expressions using the Node REPL
+- Predict the result of evaluating an expression in JavaScript
+- Define a function
+- Define a function call
+- Define the term “implementation opaque”
+- Identify the key features of a function call expression: identifier; parentheses, argument
+- Run and print the output of some basic JavaScript functions
+- Execute a JavaScript file using Node
+- Log JavaScript expressions to the terminal
+- Predict the console output when a basic JS program is executed using Node
+
+## Key vocabulary
+
+terminal, console, function, call, invoke, argument, parentheses, REPL, evaluate, implementation opaque
+
+## Notes
+
+### 💻 Interacting with computers
+
+Modern computers are highly complex devices: itt would be too difficult and time-consuming to list all the components that make up a modern computer. To begin understanding, we can start with a simple definition of a computer:
+
+{{<note title="Definition" type="info">}}
+
+A computer is a device used to store and process data. Computers have the capacity to store data through their memory and the ability to change this data through their processing ability.
```

Comment:
We're using slightly inconsistent language here vs the "Writing computer instructions" section: here we refer to "processing" and there we refer to "operations".

Might be nice to make those consistent?